bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -1189,7 +1189,18 @@ public int hashCode() {
 		 * Indicates an person's address filled in a web form
 		 */
 		TSK_WEB_FORM_ADDRESS (50, "TSK_WEB_FORM_ADDRESSES ",  //NON-NLS
-				bundle.getString("BlackboardArtifact.tskWebFormAddresses.text"));
+				bundle.getString("BlackboardArtifact.tskWebFormAddresses.text")),
```

Comment:
Should you be using the @message annotation instead of adding to the .properties file?

Reply:
Not in this case.  For consistency, we will keep all the artifact definition strings in the properties file.  I discussed it with Richard, and he and agrees.